_templates/_page_openshift.html.erb

```diff
@@ -176,7 +177,8 @@
   <script src="https://docs.openshift.com/container-platform/4.1/_javascripts/hc-search.js" type="text/javascript"></script>
   <script src="https://docs.openshift.com/container-platform/4.1/_javascripts/page-loader.js" type="text/javascript"></script>
   <script src="https://cdnjs.cloudflare.com/ajax/libs/clipboard.js/2.0.6/clipboard.min.js" type="text/javascript"></script>
-  <script src="https://docs.openshift.com/container-platform/4.1/_javascripts/clipboard.js" type="text/javascript"></script>
+  <!--<script src="https://docs.openshift.com/container-platform/4.1/_javascripts/clipboard.js" type="text/javascript"></script>REVERT-->
```

Comment:
What is the reason why these lines refer specifically to 4.1? Does this correspond with the inclusion of branchenterprise-4.1 in the labels in this PR?

Reply:
> What is the reason why these lines refer specifically to 4.1? Does this correspond with the inclusion of branchenterprise-4.1 in the labels in this PR?

@pneedle-rh 
As in my other comment, the template pulls from the 4.1 stylesheet and javascript files. In order to test updates to .js or .css files locally, you have to change the reference to your stylesheet locally. This section always gets reverted back to how it was after testing is complete and before the .css or .js updates are merged. If not, you break docs.openshift (which I have done in the past, briefly haha).

Reply:
Understood. Thanks for the clarification!